src/@ndlib/gatsby-theme-marble/components/Pages/IndexPage/index.js

```diff
@@ -60,7 +60,7 @@ const IndexPage = ({ location }) => {
   return (
     <React.Fragment>
       <BaseStyles>
-        <p dangerouslySetInnerHTML={{ __html: t('common:hompageDescriptive') }} />
+        <p sx={{ margin: '3rem' }} dangerouslySetInnerHTML={{ __html: t('common:hompageDescriptive') }} />
```

Comment:
I think the intention here is to set a maximum text block width for accessibility. So this needs to be a `maxWidth` and it probably needs to be different at the different breakpoints. Maybe something like `maxWidth: ['100%', '80rem', '80rem']` (Spitballing numbers here. I think I remember maximum accessible text width was around 70 or 80 characters wide.)

Thoughts @jonhartzler?

Reply:
yes, this should be max-width.  

WCAG standards have a maximum width a block of text should be for readability and if we put blocks of text larger than that one the site we are in violation of the standard.  So we need a max-width value not just a margin